client/my-sites/plan-features/index.jsx

```diff
@@ -70,14 +71,22 @@ class PlanFeatures extends Component {
 
 PlanFeatures.propTypes = {
 	onUgradeClick: PropTypes.func,
-	plan: React.PropTypes.oneOf( [ PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS ] ).isRequired
+	// either you specify the plan prop or isPlaceholder prop
```

Comment:
I think we would. Look at this commit, please: https://github.com/Automattic/wp-calypso/pull/6119/commits/8621b60848659cd423fb81346892216e90cd54b6, especially the `isLoadingSitePlans` const and `renderPlanPlaceholders` method. We need to check a few conditions before we can say the `PlanFeatures` is a placeholder. I think `PlanFeatures` don't have access to that data.